src/main/java/HUPPAAL.java

```diff
@@ -1,6 +1,9 @@
+import dk.cs.aau.huppaal.BuildConfig;
+
 public class HUPPAAL {
+    //For JavaFX Runtime Components are Missing https://edencoding.com/runtime-components-error/ - 2021-02-09
 	public static void main(String[] args) {
-	    //For for JavaFX Runtime Components are Missing https://edencoding.com/runtime-components-error/ - 2021-02-09
+        System.out.printf("welcome to %s version v%s+%s%n", BuildConfig.NAME, BuildConfig.VERSION, BuildConfig.COMMIT_SHA_SHORT);
```

Comment:
Capitalize starting letter, but good idea with a greeting :+1: It has just been added to ECDAR as well

Reply:
Sounds good :smile: 

Capitalized in a2b3c9c4872fc881c8b6a119fe9f8e8dc41d41a2